management/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM gcr.io/distroless/base:debug
+FROM gcr.io/distroless/base
 ENTRYPOINT [ "/go/bin/wiretrustee","management"]
```

Comment:
I thought it would be a separate binary, no?

Reply:
Looks like it is @andpar83 
When I run `goreleaser --debug --snapshot --skip-publish --rm-dist` locally it produces a folder `dist/` with separate folders and each of them has a `wiretrustee` binary in it. 

├── goreleaserdocker332486772
├── goreleaserdocker417631529
├── goreleaserdocker460708675
├── goreleaserdocker464271775
├── goreleaserdocker819584114
├── wiretrustee_darwin_amd64
├── wiretrustee_linux_amd64
├── wiretrustee_linux_arm_6
├── wiretrustee_linux_arm64
**├── wiretrustee-mgmt_linux_amd64
├── wiretrustee-mgmt_linux_arm64**
└── wiretrustee_windows_amd64

When I run this:
`./dist/wiretrustee-mgmt_linux_amd64/wiretrustee --help` I get only management service-related output. 
Seems to be correct, right @mlsmaycon ?
```
Usage:
  wiretrustee [command]

Available Commands:
  help        Help about any command
  management  start Wiretrustee Management Server

Flags:
      --config string      Wiretrustee config file location to write new config to (default "/etc/wiretrustee/config.json")
  -h, --help               help for wiretrustee
      --log-level string    (default "info")

Use "wiretrustee [command] --help" for more information about a command.

```

Reply:
I am able to generate a different binary as well. Do you guys prefer that?

Reply:
@mlsmaycon yeah, I think this way is better